scripts/exglobal_atmos_products.sh

```diff
@@ -194,6 +198,14 @@ if [[ "${FLXGF:-}" == "YES" ]]; then
   done
 fi
 
+# Section creating 0.25 degree WGNE products for nset=1, and fhr <= FHMAX_WGNE
+if [[ "${WGNE:-}" == "YES" ]]; then
+  grp=""  # TODO: this should be "a" when we eventually rename the pressure grib2 files per EE2 convention
+  if (( FORECAST_HOUR > 0 & FORECAST_HOUR <= FHMAX_WGNE )); then
+    ${WGRIB2} "${COM_ATMOS_GRIB_0p25}/${PREFIX}pgrb2${grp}.0p25.${fhr3}" -d 597 -grib "${COM_ATMOS_GRIB_0p25}/${PREFIX}wgne.${fhr3}"
```

Comment:
I actually like the compact `-d 597` rather than the pipe and pipe.  
But thank you for providing the info on what is being done here, I'll add that blurb to document it.

Reply:
@aerorahul The record 597 is hard-wired code for GFS V16 product only. That record number will be changed with new variables added for GFS V17. You might add the pipe and pipe way as workaround and switch to record number when GFS v17 code is finalized.

Reply:
Great. We can set `WGNE=NO` for now and update the record number later.

Reply:
I have added a configuration option while this gets worked on.  We will make firm when the science code freeze occurs.